src/terra-dev-site/contributing/GitWorkflow.b.contributing.md

```diff
@@ -0,0 +1,112 @@
+# Git Workflow
+The follow section outlines our Git workflow for Terra UI projects.
+
+## Fork The Repository
+
+Start by [forking](https://help.github.com/articles/fork-a-repo/) the repo you’ll be working on.
+
+_Note: The [terra team](https://github.com/orgs/cerner/teams/terra) and [terra contributors](https://github.com/orgs/cerner/teams/terra-contributors) can ignore this step and clone the repo directly._
+
+## Clone The Repo
+
+Example:
+
+`git clone git@github.com:cerner/terra-core.git`
+
+## Clone Your Forked Repo
+
+New contributors will need to clone their forked repo.
+
+Example:
+
+`git clone git@github.com:<github-username>/terra-core.git`
+
+## Creating A Branch
+
+Before starting your work, first ensure you're in the master branch and that you've pulled down the latest changes:
+
+```sh
+git checkout master
+git pull
+```
+
+Next, create a new branch for your work, with an appropriate name for your change:
+
+`git checkout -b my-feature-or-bug-branch`
+
+## Development Workflow
+
+Read through our [development workflow guide](#/contributing/terra-ui/developer-workflow) to better understand the development process.
+
+## Adding, Committing, And Pushing Changes
+
+Regularly git commit locally and git push to the remote branch. [Write good commit messages](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
+Your commits will automatically be squashed when your PR is merged into master.
+
+```sh
+git add .
+git commit -m "Fixes issue #123"
+```
+
+This will commit your changes to your local copy of the repo.
+
+Then push your changes to the remote repo.
+
+## Submit A PR
+
+When your branch is ready, open a new pull request via GitHub. The repo [PR template](https://github.com/cerner/terra-ui/blob/master/.github/PULL_REQUEST_TEMPLATE.md) will guide you toward describing your contribution.
+
+You can [use words in PR description that refrence the related issue](https://help.github.com/articles/closing-issues-using-keywords/) and will automatically close it when the PR is merged.
+
+For example, to close an issue numbered `123`, you could use the phrase `Closes #123` or `Closes: #123` in your pull request description.
+
+## Code Review
+
+Per our [terra governance model](https://github.com/cerner/terra-ui/blob/master/GOVERNANCE.md), changes must be reviewed by the maintainers. This includes contributions from the community as well as the current maintainers. We use [Github pull requests](https://help.github.com/articles/about-pull-requests/) and [Github pull request reviews](https://help.github.com/articles/about-pull-request-reviews/) to manage reviewing and merging the proposed changes.
```

Comment:
This includes contributions from the community **add comma** as well as the current maintainers.

Reply:
Updated: https://github.com/cerner/terra-ui/pull/108/commits/453b149ee491fa9a8d004c85dc550ca83dedf36d